sdk/storage/storage-blob/src/utils/constants.ts

```diff
@@ -50,3 +50,8 @@ export const HeaderConstants = {
 
 export const ETagNone = "";
 export const ETagAny = "*";
+
+export const REQUIRE_DFS_ENDPOINT_FLAG = "REQUIRE_DFS_ENDPOINT_FLAG";
+
+export const BLOB_SERVICE_PUBLIC_CLOUD_HOST = "blob.core.windows.net";
```

Comment:
Haven't check everywhere using this const, Please ensure this would not cause SDK only support public cloud.

Reply:
Should be

---
In reply to: [318948382](https://github.com/Azure/azure-sdk-for-js/pull/4874#discussion_r318948382) [](ancestors = 318948382)